services/catalog/requirements/_test.txt

```diff
@@ -1,180 +1,115 @@
 aiohttp==3.9.3
-    # via
-    #   -c requirements/../../../requirements/constraints.txt
-    #   -c requirements/_base.txt
-    #   pytest-aiohttp
+    # via pytest-aiohttp
 aiosignal==1.3.1
-    # via
-    #   -c requirements/_base.txt
-    #   aiohttp
+    # via aiohttp
 alembic==1.13.1
-    # via
-    #   -c requirements/_base.txt
-    #   -r requirements/_test.in
 anyio==4.3.0
-    # via
-    #   -c requirements/_base.txt
-    #   httpx
+    # via httpx
+asgi-lifespan==2.1.0
 async-timeout==4.0.3
-    # via
-    #   -c requirements/_base.txt
-    #   aiohttp
+    # via aiohttp
 attrs==23.2.0
     # via
-    #   -c requirements/_base.txt
     #   aiohttp
     #   jsonschema
     #   pytest-docker
     #   referencing
 certifi==2024.2.2
     # via
-    #   -c requirements/../../../requirements/constraints.txt
-    #   -c requirements/_base.txt
     #   httpcore
     #   httpx
     #   requests
 charset-normalizer==3.3.2
     # via requests
 click==8.1.7
```

Comment:
aren't you using an old uv here? why are all the via going away?

Reply:
yes i was. thx!